### PR TITLE
Use compass add in agile and codegen recipes

### PR DESCRIPTION
**Pull request by mcraveiro**
## Summary

- `how_do_i_plan_a_sprint.org`: promote story scaffold step to use `compass.sh add story` (auto-defaults to current sprint); keep `--parent-dir` as an explicit override note
- `how_do_i_open_a_new_sprint.org`: scaffold sprint via `compass.sh add sprint` (auto-defaults to current version); update `* Script` section accordingly
- `how_do_i_refine_the_backlog.org`: file new captures via `compass.sh add capture`
- `how_do_i_create_a_new_v2_doc.org`: add a prominent note at the top of the Answer section directing story/task/sprint work to `compass add`; retain full codegen reference for all other doc types and edge cases

In all updated recipes the `* See also` section now links to [[How do I add a doc with compass?]] as the preferred entry point.

## Test plan

- [ ] Verify each updated recipe renders cleanly on the site
- [ ] Smoke-test `compass.sh add story`, `add sprint`, `add capture` to confirm they accept the flags shown in the recipes

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: doc/recipes/agile/how_do_i_open_a_new_sprint.org [hunk 1]
```diff
@@ -37,14 +37,14 @@ folder, set its mission, and link it from the version manifest?
...
+3. /Scaffold the sprint/. Use =compass add= — it defaults =--parent-dir=
+   to the current version automatically and mints a fresh =:ID:=,
+   applies the right frontmatter, builds the skeleton sections, and the
    ancestor tag chain:
 
    #+begin_src sh :results verbatim
-   projects/ores.codegen/generate_v2_doc.sh \
-     --type sprint --slug sprint_17 \
-     --parent-dir doc/agile/versions/v0 \
+   ./projects/ores.compass/compass.sh add sprint \
```
[the remark this answers is not in the document]
Comment (mcraveiro) [reply doc/recipes/agile/how_do_i_open_a_new_sprint.org]: Accepted. Removed `./` prefix from all three `compass.sh add` invocations in 1e21894f3.

File: doc/recipes/agile/how_do_i_open_a_new_sprint.org [hunk 2]
```diff
@@ -77,7 +77,8 @@ folder, set its mission, and link it from the version manifest?
 
 * Script
 
-The wrapper is =projects/ores.codegen/generate_v2_doc.sh=. The
+=projects/ores.compass/compass.sh add= → =src/compass.py= (=cmd_add=),
```
[the remark this answers is not in the document]
Comment (mcraveiro) [reply doc/recipes/agile/how_do_i_open_a_new_sprint.org]: Accepted. Rephrased in 1e21894f3 to: "The entry point is `projects/ores.compass/compass.sh add`, mapping to `src/compass.py` (`cmd_add`), which calls `ores.codegen`'s `v2_doc_generate` as a library."

File: doc/recipes/agile/how_do_i_plan_a_sprint.org [hunk 3]
```diff
@@ -57,16 +57,19 @@ against it, and promote selected captures into stories?
...
 
    a. Scaffold the story in the sprint folder, preserving the
-      capture's =:ID:= with =--id=:
+      capture's =:ID:= with =--id=. Use =compass add= — it defaults
+      =--parent-dir= to the current sprint automatically:
 
       #+begin_src sh :results verbatim
-      projects/ores.codegen/generate_v2_doc.sh \
-        --type story --slug <slug> \
+      ./projects/ores.compass/compass.sh add story \
```
[the remark this answers is not in the document]
Comment (mcraveiro) [reply doc/recipes/agile/how_do_i_plan_a_sprint.org]: Accepted. Fixed in 1e21894f3.

File: doc/recipes/agile/how_do_i_refine_the_backlog.org [hunk 4]
```diff
@@ -48,12 +48,14 @@ sharpening vague ones, and re-prioritising between =near/= and
 3. /File new captures/ that surface during the review:
 
    #+begin_src sh :results verbatim
-   projects/ores.codegen/generate_v2_doc.sh \
-     --type capture --slug <slug> \
+   ./projects/ores.compass/compass.sh add capture \
```
[the remark this answers is not in the document]
Comment (mcraveiro) [reply doc/recipes/agile/how_do_i_refine_the_backlog.org]: Accepted. Fixed in 1e21894f3.
